test/dynamo/test_higher_order_ops.py

```diff
@@ -227,6 +227,26 @@ def f(x):
             f, (x,), ifdynstaticdefault(2, 3), expected_opcount=ifdynstaticdefault(1, 2)
         )
 
+    def test_wrap_pytree_args(self):
+        def f(x, y):
+            return wrap(lambda z: z[0].sin() * z[1].cos(), (x, y))
+
+        x = torch.tensor(1.5)
+        y = torch.tensor(2.0)
+        self._test_wrap_simple(f, (x, y), 3)
```

Comment:
Is it possible to get the graph output and do a self.assertExpectedInline on it? We want to make sure that the graph input is flattened.